src/doc/gui_manual/Operators/OperatorTypes/PICS/LCS_operator.rst

```diff
@@ -0,0 +1,183 @@
+.. _LCS operator:
+
+Lagrangian Coherent Structure operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The LCS operator utilizes Lyapunov Exponents based on the Cauchy-Green Tensor
+to highlight LCS's in vector fields. The Cauchy-Green tensor is used as an
+equivalent definition because it is independent of the mesh discretization. As
+such, when performing an FTLE calculation, the time can be specified as one
+would for a traditional FTLE, and the resulting value will be based on the
+maximal Eigen value.
+
+However, when performing the calculate with Finite Space Lyapunov Exponents
+(FSLE), instead of assuming a uniform mesh discretization and specifying the
+dispersion distance, we specify a dispersion factor which in a traditional FTLE
```

Comment:
`...we specify a dispersion factor which in a traditional` -> `...we specify a dispersion factor. In a traditional FTLE, this is the...`